po/extra/pt.po

```diff
@@ -0,0 +1,360 @@
+"are no packages in the folder"
+msgstr ""
+"Corrige a opção de escolha \"Carregar das Transferências\" de apenas "
+"mostrar um botão giratório se não houver pacotes na pasta"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:118
+msgid "More debug information in the app output if ran in a shell"
+msgstr ""
+"Mais informação de depuração na saída da aplicação se executada numa shell"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:119
+msgid "Rename executable to match RDNN naming convention"
+msgstr "Renomear o executável para corresponder à convenção de nomeação RDNN"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:126
+msgid "Add a button to update / downgrade already installed package"
+msgstr "Adiciona um botão para atualizar / desatualizar o pacote já instalado"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:127
+msgid "Fix window growing in size when summary or package name is really long"
+msgstr ""
+"Corrige o aumento do tamanho da janela quando o resumo ou nome do pacote é "
+"muito comprido"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:128
+msgid "The application can be now found under \"System\" category"
+msgstr "A aplicação pode agora ser encontrada na categoria \"Sistema\""
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:129
+msgid "Ctrl-Q now exists the app"
+msgstr "Ctrl-Q agora faz sair da aplicação"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:136
+msgid "Do not display error dialog when operation cancelled"
+msgstr "Não mostra a caixa de diálogo de erro quando a operação é cancelada"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:137
+msgid "Display the progress of current operation in the dock"
+msgstr "Mostrar o progresso da operação atual na Doca"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:138
+msgid "Sort packages by their state and name in the list view"
+msgstr "Ordena os pacotes por estado e nome na visualização de lista"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:145
+msgid "Save window state between sessions"
+msgstr "Guardar o estado da janela entre as sessões"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:146
+msgid "Improved performance of launching the app second time"
+msgstr "Melhor desempenho ao iniciar a aplicação pela segunda vez"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:153
+msgid "Support other package formats such as .rpm for other package managers"
+msgstr ""
+"Suporta outros formatos de pacote como .rpm para outros gestores de pacotes"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:154
+msgid "Minor fixes"
+msgstr "Pequenas correções"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:161
+msgid "Added an ability to open packages with Eddy in the file manager"
+msgstr ""
+"Adicionada a capacidade de abrir pacotes com o Eddy no gestor de ficheiros"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:162
+msgid "Show a spinner when loading packages"
+msgstr "Mostrar um botão giratório ao carregar pacotes"
```

Comment:
Shouldn't this be "Mostra", for verb tense consistency with the other entries?

Reply:
I try to make the translations as formal as possible but in this case I believe that @waldyrious is right.
I'll fix that.